tests/Extensions/Abstractions.UnitTests/ValidationTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Omex.Extensions.Abstractions.UnitTests
+{
+	[TestClass]
+	public class ValidationTests
+	{
+		[TestMethod]
+		public void ThrowIfNullOrWhiteSpace_WhenValueValidRefType_ReturnsIt()
+		{
+			object expected = new object();
+			object actual = Validation.ThrowIfNull(expected);
+			Assert.AreEqual(expected, actual);
+		}
+
+		[TestMethod]
+		public void ThrowIfNullOrWhiteSpace_WhenValueValidStruct_ReturnsIt()
```

Comment:
These two tests should be ThrowIfNull_...

Reply:
Thank you, too much copy/paste